app/models/foreman_virt_who_configure/config.rb

```diff
@@ -247,6 +248,18 @@ def status_description
       _(STATUS_DESCRIPTIONS[status])
     end
 
+    def proxy_type
```

Comment:
Seems like this method could go in the output_generator.rb since it deals with the environment variables that get set